lib/createStylelintResult.js

```diff
@@ -1,5 +1,5 @@
-/* @flow */
 "use strict"
+/* @flow */
```

Comment:
@davidtheclark hm..actually I do not know why it becomes like it is now... I'm pretty sure I could not make it... e.g. `needlessDisables.js` I add `// @flow` above as it should be... anyway I'll fix that, my bad